src/app/(proper_react)/Shell.stories.tsx

```diff
@@ -17,7 +18,9 @@ type Story = StoryObj<typeof ShellEl>;
 export const Shell: Story = {
   render: () => (
     <ShellEl l10n={getL10n()} session={null}>
-      <div style={{ height: 800 }}></div>
+      <div style={{ height: 800, backgroundColor: "#e7e7e7" }}>
```

Comment:
Not sure if we can use colors from tokens.scss here or not, but&hellip; 🤷 

```css
$color-grey-10: #e7e7e7;
```

Reply:
Unfortunately we can't :/ Luckily this is limited to the demo, so if the colour is off that shouldn't matter too much.